libflux/flux-core/src/formatter/mod.rs

```diff
@@ -32,6 +32,15 @@ pub fn convert_to_string(file: &File) -> Result<String, Error> {
 }
 
 /// Format a string of Flux code.
+///
+/// # Example
+///
+/// ```rust
+/// # use fluxcore::formatter::format;
+/// let source = "(r) => r.user ==              \"user1\"";
```

Comment:
Did the formatter do this?

Reply:
This is intentional so that we can run `fluxcore::formatter::format`on `source` and get the formatted Flux code.

Reply:
The `#` is to hide boilerplate code in the rendered docs. But perhaps the `use` statement would be nice to show here...

Reply:
Yeah, sorry. The reason I asked is that we have a current conversation going on around horizontal alignment, so wondered if we had another case.